sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -286,13 +288,14 @@ object Webpack {
   /**
     * Runs the webpack command.
     *
+    * @param nodeFlags node.js cli flags
     * @param args Arguments to pass to the webpack command
     * @param workingDir Working directory in which the Nodejs will be run (where there is the `node_modules` subdirectory)
     * @param log Logger
     */
-  def run(args: String*)(workingDir: File, log: Logger): Option[WebpackStats] = {
+  def run(nodeFlags: String*)(args: String*)(workingDir: File, log: Logger): Option[WebpackStats] = {
     val webpackBin = workingDir / "node_modules" / "webpack" / "bin" / "webpack"
-    val cmd = Seq("node", webpackBin.absolutePath, "--bail", "--profile", "--json") ++ args
+    val cmd = Seq("node") ++ nodeFlags ++ Seq(webpackBin.absolutePath, "--bail", "--profile", "--json") ++ args
```

Comment:
We should then simplify the expression and not create a singleton sequence containing `"node"`.